examples/kubernetes/cross_account_mount/README.md

```diff
@@ -21,14 +21,17 @@ parameters:
   gidRangeEnd: "2000"
   basePath: "/dynamic_provisioning"
   az: us-east-1a
+  csi.storage.k8s.io/provisioner-secret-name: x-account
+  csi.storage.k8s.io/provisioner-secret-namespace: kube-system
 ```
 
 ### Prerequisite setup
 1. Perform [vpc-peering](https://docs.aws.amazon.com/vpc/latest/peering/working-with-vpc-peering.html) between EKS cluster `vpc` in aws account `A` and EFS `vpc` in another aws account `B` .
 2. Create an IAM role in Account `B` which has a trust relationship with Account `A` and add an EFS policy with permissions to call `DescribeMountTargets`. This role will be used by CSI-Driver to determine the mount targets for file system in account `B`
```

Comment:
since we are providing an example of the assumerole policy, can we provide one for this as well?